lib/v2/magazinelib/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'magazinelib.com': {
+        _name: 'Magazine',
```

Comment:
```suggestion
        _name: 'MagazineLib',
```
`_name` should follow H2 header